### DISCUSSION TITLE
数据更新速度

### DISCUSSION THREAD

**Discussion by AlinaBaby2019** [WeNeedHome/SummaryOfLoanSuspension#1004] [Q&A]
大家好:
    目前看数据更新有些慢，是否已经到达顶峰，接近全国所有数据了？还是有某方阻碍导致数据不能更新了？

**Comment by sqpk** [comment]
热度问题，另外烂尾楼选择停贷的数量快到极限了，应该还有大部分选择不停贷，全国烂尾楼盘应该有上千吧

**Reply by imhappyok** [reply]
上千，你太小看厉害国了，我们小县城都好几个烂尾楼，厉害国几千个县城，你要算一算吗？
